python practice/day17/solution.py

```diff
@@ -3,6 +3,7 @@ class Calculator:
     def power(n,p):
         if p < 0 or n < 0:
             print('n and p should be non-negative')
+            raise ValueError('A very specific bad thing happened.')
         z = 1 
         for i in range(0,p):
```

Comment:
python have a operator for this power operation: **
for example: 2**4 = 16

Reply:
Right. I am sorry , I think it's like c++ which there is no power operator.